net/nginx/Makefile

```diff
@@ -186,8 +193,12 @@ ifneq ($(CONFIG_NGINX_HTTP_UPSTREAM_KEEPALIVE),y)
   ADDITIONAL_MODULES += --without-http_upstream_keepalive_module
 endif
 
+TARGET_CFLAGS += -fvisibility=hidden -ffunction-sections -fdata-sections -DNGX_LUA_NO_BY_LUA_BLOCK
```

Comment:
For the reason behind the `-fvisibility=hidden` see http://mailman.nginx.org/pipermail/nginx-devel/2016-March/008010.html